core_backend/app/users/routers.py

```diff
@@ -258,7 +260,9 @@ async def create_first_user(
     # 1.
     user.role = UserRoles.ADMIN
     user.workspace_name = user.workspace_name or f"{user.username}'s Workspace"
-    workspace_db_new, _ = await create_workspace(asession=asession, user=user)
+    workspace_db_new, _ = await create_workspace(
+        asession=asession, doc_language=IdentifiedLanguage.ENGLISH, user=user
```

Comment:
For the very first user, the default document language is `ENGLISH`. the user can then update this in the UI after logging in.